tools/expotools/src/commands/ClientBuild.ts

```diff
@@ -0,0 +1,125 @@
+import path from 'path';
+import fs from 'fs-extra';
+import chalk from 'chalk';
+import aws from 'aws-sdk';
+import inquirer from 'inquirer';
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import { Config, UpdateVersions } from '@expo/xdl';
+
+import { Platform, iosAppVersionAsync, androidAppVersionAsync } from '../ProjectVersions';
+import { STAGING_HOST, EXPO_DIR, IOS_DIR, ANDROID_DIR } from '../Constants';
+import askForPlatformAsync from '../utils/askForPlatformAsync';
+import getSDKVersionFromBranchNameAsync from '../utils/getSDKVersionFromBranchNameAsync';
+
+type ActionOptions = {
+  platform?: 'ios';
+  release: boolean;
+};
+
+async function askToRecreateSimulatorBuildAsync(archivePath: string) {
+  if (process.env.CI) {
```

Comment:
I would prefer not calling this function if in CI (instead of early return inside this function). If you think it is really better approach, let's `console.log` something that we skip asking in CI (for debugging purposes)

Reply:
In most use cases I'd agree on adding a console.log for debugging purposes, but here it probably doesn't make much sense since this method is invoked only if the build (made by `client_ios` job) is still in cache - which is gonna be true in most cases. Also, appropriate log is printed a bit later, see https://github.com/expo/expo/pull/5331/files#diff-e6957a290b001e537366cb96c4143de8R76

Reply:
Sure, I understand the reasons why the `console.log` shouldn't ever be needed, I just don't like such mechanisms 🙂 🤷‍♂